src/actions/navigation.js

```diff
@@ -11,6 +11,9 @@ import { errorNotification } from "./notifications";
  */
 export const chooseDisplayComponentFromURL = (url) => {
   const parts = url.toLowerCase().replace(/^\/+/, "").replace(/\/+$/, "").split("/");
+  // todo - use URL() not the above code, but `url` is not actually the URL so...
```

Comment:
It's just the pathname (well, it looks like the pathname). E.g. `"flu/seasonal/h3n2/na/2y"`. It would be better to pass around `URL()` objects I think.